examples/step-44/doc/intro.dox

```diff
@@ -109,29 +109,29 @@ such that
 		\qquad \text{and} \qquad
 	\dfrac{1}{2}[\mathbf{A} - \mathbf{A}^T] = \mathcal{W}:\mathbf{A} \, .
 @f]
-The fourth-order <code>SymmetricTensor</code> returned by identity_tensor() is $\mathcal{S}$.
+The fourth-order <code>SymmetricTensor</code> returned by identity_tensor() is $\mathcal{I}$.
 
 
 <h3>Kinematics</h3>
 
 Let the time domain be denoted $\mathbb{T} = [0,T_{\textrm{end}}]$, where $t \in \mathbb{T}$ and $T_{\textrm{end}}$ is the total problem duration.
 Consider a continuum body that occupies the reference configuration $\Omega_0$ at time $t=0$.
-Particles in the reference configuration are identified by the position vector $\mathbf{X}$.
+%Particles in the reference configuration are identified by the position vector $\mathbf{X}$.
 The configuration of the body at a later time $t>0$ is termed the current configuration, denoted $\Omega$, with particles identified by the vector $\mathbf{x}$.
 The nonlinear map between the reference and current configurations, denoted $\boldsymbol{\varphi}$, acts as follows:
 @f[
 	\mathbf{x} = \boldsymbol{\varphi}(\mathbf{X},t) \, .
 @f]
 The material description of the displacement of a particle is defined by
 @f[
-	\mathbf{U}(\mathbf{X},t) = \mathbf{x}(\mathbf{X},t) - \mathbf{X} \, .
+	\mathbf{U}(\mathbf{X},t) = \boldsymbol{\varphi}(\mathbf{X},t) - \mathbf{X} \, .
 @f]
 
 The deformation gradient $\mathbf{F}$ is defined as the material gradient of the motion:
 @f[
 	\mathbf{F}(\mathbf{X},t)
 		\dealcoloneq \dfrac{\partial \boldsymbol{\varphi}(\mathbf{X},t)}{\partial \mathbf{X}}
-		= \textrm{Grad}\ \mathbf{x}(\mathbf{X},t)
+		= \textrm{Grad}\ \mathbf{\varphi}(\mathbf{X},t)
```

Comment:
This now reads exactly the same as the previous line?

Reply:
I thought the point of this sentence was to introduce `Grad`, but given your other point on notation I will change it back.

Reply:
I can only presume that I meant to introduce `Grad` and work back towards the link between current position and displacement.

Reply:
Thank you for giving these points some thought :-)